text-minimessage/src/test/java/net/kyori/adventure/text/minimessage/MiniMessageSerializerTest.java

```diff
@@ -161,7 +171,7 @@ void testContinuedClick() {
 
   @Test
   void testKeyBind() {
-    final String expected = "Press <key:key.jump> to jump!";
+    final String expected = "Press <key:key.jump></key> to jump!";
```

Comment:
Is this required now? Or just done for explicitness?

Reply:
thats just what the new serializer produces, since this tag is just inserting a component, the old behavior was "nicer" I guess?

Reply:
It's the difference between " to jump" being a child of the keybind component, and "to jump" being a sibling of the keybind component -- self-closing tags will make this component layout nicer.